packages/salesforcedx-vscode-core/src/commands/auth/forceAuthAccessTokenLogin.ts

```diff
@@ -46,8 +52,10 @@ export class ForceAuthAccessTokenExecutor extends LibraryCommandletExecutor<
       await authInfo.save();
       await authInfo.setAlias(alias);
       await authInfo.setAsDefault({
-        defaultUsername: true
+        org: true,
+        devHub: true
```

Comment:
Its seems like parity with previous functionality here would be to only set org to true, since only default username (and not default dev hub username) was being set before.

Reply:
Good catch @klewis-sfdc! Will be taking this back out in my PR 👍